plugins/apps/clementine.mk

```diff
@@ -0,0 +1,79 @@
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgstgdp.dll'               '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgstgio.dll'               '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgsticydemux.dll'          '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgstid3demux.dll'          '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgstisomp4.dll'            '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgstlame.dll'              '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgstmad.dll'               '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgstogg.dll'               '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgstopus.dll'              '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgstplayback.dll'          '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgstreplaygain.dll'        '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgstsouphttpsrc.dll'       '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgstspectrum.dll'          '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgstspeex.dll'             '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgsttaglib.dll'            '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgsttcp.dll'               '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgsttypefindfunctions.dll' '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgstudp.dll'               '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgstvolume.dll'            '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgstvorbis.dll'            '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgstwavpack.dll'           '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)/bin/libgstwavparse.dll'          '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
```

Comment:
I see. Can you avoid copy-paste of the install command:

```
$(INSTALL) '$(PREFIX)/$(TARGET)'/bin/{libgstvolume,libgstvorbis,libgstwavpack,...}.dll '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
```

?

(Ideally this list of files could be generated from packages name, but there is no way to do it currently.)